test-utils/src/main/java/apoc/util/TestContainerUtil.java

```diff
@@ -149,6 +149,7 @@ private static Neo4jContainerExtension createNeo4jContainer(List<ApocPackage> ap
                 .withNeo4jConfig("dbms.logs.http.enabled", "true")
                 .withNeo4jConfig("dbms.logs.debug.level", "DEBUG")
                 .withNeo4jConfig("dbms.routing.driver.logging.level", "DEBUG")
+                .withNeo4jConfig("internal.dbms.type_constraints", "true")
```

Comment:
Why do we have integration tests for this instead of unit tests?

Reply:
Well they are enterprise edition only features, so thought they needed to be done this way due to that? I got errors at least when trying with unit tests, and didn't try much more than that. Is there a way to do them that you know of?

Reply:
Ahhhh, I overlooked DDL commands are enterprise only. All good 👍